sh.funkplot.Funkplot.yml

```diff
@@ -0,0 +1,22 @@
+app-id: sh.funkplot.Funkplot
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+base: "io.qt.qtwebengine.BaseApp"
+base-version: "5.15-22.08"
+command: funkplot-gui
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --device=dri
```

Comment:
also missing `cleanup` rules to remove files not needed at build time like the headers and the cmake files.